common/religion/holy_sites/wc_holy_sites.txt

```diff
@@ -1180,6 +1180,17 @@ taunkale = {
 #njord
 #grizzlemaw
 
+### Trogg religion
+alterac_valley = {
+	county = c_alterac_valley
+
+	character_modifier = {
+		name = holy_site_alterac_valley_effect_name
+		martial = 1
+		monthly_prestige_gain_per_knight_add = 0.1
```

Comment:
Balanced it [Here](https://docs.google.com/spreadsheets/d/18zS-FAte6nsejx7rpzSmVuioRXps3iWSvSkx_xRJSWI/edit?usp=sharing).
```suggestion
		martial = 1
		monthly_prestige_gain_per_knight_add = 0.2
```